src/platform/qt/InputProfile.h

```diff
@@ -48,15 +48,9 @@ class InputProfile {
 
 	constexpr InputProfile(const char* name,
 	                       int keys[GBA_KEY_MAX],
-	                       const ShortcutButton* shortcutButtons = (ShortcutButton[]) {{}},
-	                       const ShortcutAxis* shortcutAxes = (ShortcutAxis[]) {{}},
-	                       AxisValue axes[GBA_KEY_MAX] = (AxisValue[GBA_KEY_MAX]) {
-	                       	                             {}, {}, {}, {}, 
-	                                                     { GamepadAxisEvent::Direction::POSITIVE, 0 },
-	                                                     { GamepadAxisEvent::Direction::NEGATIVE, 0 },
-	                                                     { GamepadAxisEvent::Direction::NEGATIVE, 1 },
-	                                                     { GamepadAxisEvent::Direction::POSITIVE, 1 },
-	                                                     {}, {}},
+						   const ShortcutButton* shortcutButtons,
+						   const ShortcutAxis* shortcutAxes,
+						   AxisValue axes[GBA_KEY_MAX],
```

Comment:
I took this out because I was getting
> error C4576: a parenthesized type followed by an initializer list is a non-standard explicit type conversion syntax